pages/common/[.md

```diff
@@ -2,24 +2,24 @@
 
 > Check file types and compare values.
 > Returns 0 if the condition evaluates to true, 1 if it evaluates to false.
-> More information: <https://www.gnu.org/software/coreutils/test>.
+> More information: <https://www.gnu.org/software/bash/manual/bash.html#Conditional-Constructs>.
 
-- Test if a given variable is equal to a given string:
+- Test if two strings are equal/not equal:
```

Comment:
The original makes more sense since you are moving the $ outside the {{}}

Reply:
I've updated descriptions to be more user-friendly. ;)